src/ImageRunModal.jsx

```diff
@@ -152,6 +251,9 @@ export class ImageRunModal extends React.Component {
                            checked={this.state.hasTTY}
                            onChange={e => this.onValueChanged('hasTTY', e.target.checked)} />
                 </label>
+
+                <label className='control-label' htmlFor='run-image-dialog-publish'> {_("Ports")} </label>
```

Comment:
Please drop the extra space outside the { }, as they render in HTML. Instead, move it inside the { } (we usually do that).